Mage.Sets/src/mage/cards/t/ThePrincessTakesFlight.java

```diff
@@ -0,0 +1,57 @@
+package mage.cards.t;
+
+import java.util.UUID;
+
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.abilities.common.SagaAbility;
+import mage.abilities.effects.Effects;
+import mage.abilities.effects.common.ExileTargetForSourceEffect;
+import mage.abilities.effects.common.ReturnFromExileForSourceEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SagaChapter;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class ThePrincessTakesFlight extends CardImpl {
+
+    public ThePrincessTakesFlight(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{2}{W}");
+        
+        this.subtype.add(SubType.SAGA);
+
+        // (As this Saga enters and after your draw step, add a lore counter. Sacrifice after III.)
+        SagaAbility sagaAbility = new SagaAbility(this);
+
+        // I -- Exile up to one target creature.
+        sagaAbility.addChapterEffect(this, SagaChapter.CHAPTER_I, new Effects(new ExileTargetForSourceEffect()), new TargetCreaturePermanent(0, 1));
```

Comment:
(shouldn't need the Effects wrapper, constructor that takes an Effect should handle it)

Reply:
You might need to have two SagaChapter.CHAPTER_I then, because I do not think there is call for `addChapterEffect(Ability, SagaChapter, Effect, Target)`

Reply:
Oh that explains it, never mind

Reply:
Yeah, that's what happened. It was either an `Effects` wrapper or an extra `SagaChapter` parameter, and I just picked what I thought looked nicer x3